docs/integrations/ember-and-json-api.md

```diff
@@ -12,6 +13,21 @@ Note: This guide assumes that `ember-cli` is used for your ember app.
 The JSON API specification calls for hyphens for multi-word separators. AMS uses underscores.
 To solve this, in Ember, both the adapter and the serializer will need some modifications:
 
+### Server-Side Changes
+
+there are multiple mimetypes for json that should all be parsed similarly, so
+in `config/initializers/mime_types.rb`:
+```ruby
+api_mime_types = %W(
+  application/vnd.api+json
+  text/x-json
+  application/json
+)
+
+Mime::Type.unregister :json
+Mime::Type.register 'application/json', :json, api_mime_types
```

Comment:
Not sure if we might want something like https://github.com/cerebris/jsonapi-resources/blob/master/lib/jsonapi/mime_types.rb but I have no way of testing it right now

```ruby
module JSONAPI
  MEDIA_TYPE = 'application/vnd.api+json'
end

Mime::Type.register JSONAPI::MEDIA_TYPE, :api_json

ActionDispatch::ParamsParser::DEFAULT_PARSERS[Mime::Type.lookup(JSONAPI::MEDIA_TYPE)] = lambda do |body|
  data = JSON.parse(body)
  data = {:_json => data} unless data.is_a?(Hash)
  data.with_indifferent_access
end
```